# closes the figures and plots
string sessionname
int32 sessionid

# array of ids returned from orEnvPlot or other plotting functions
int32[] figureids
---
